java/timeouts_application/src/main/java/org/demo/workflow/WorkflowOutput.java

```diff
@@ -0,0 +1,28 @@
+package org.demo.workflow;
+
+public class WorkflowOutput {
```

Comment:
simpler to use a one-line record in ConductorWorkers instead
record WorkflowOutput(String phoneNumber, String email) {}

Reply:
Wanted to show a distinct output for the alerts case, and would be partly evident in other cases